internal/controller/validator_test.go

```diff
@@ -1523,8 +1574,7 @@ func TestDPAReconciler_ValidateDataProtectionCR(t *testing.T) {
 					NonAdmin: &oadpv1alpha1.NonAdmin{
 						Enable: ptr.To(true),
 						EnforceRestoreSpec: &velerov1.RestoreSpec{
-							IncludedNamespaces: []string{"banana"},
```

Comment:
currently this is allowed for restore spec

Once a future PR updates it, I would update test

Reply:
Yeah lets tackle the backup api first